dataOrganization.md

```diff
@@ -0,0 +1,62 @@
+# Firestore Data Organization
+## Users
+### Collections
+* [Feeders](#feeders)
+* [Notifications](#notifications)
+### Document Fields
+```ts
+{
+  active: boolean,
+  appIdentifier: string,
+  email: string,
+  firstName: string,
+  id: string,
+  lastName: string,
+  lastOnlineTimestamp: timestamp,
+  phoneNumber: string,
+  profilePictureURL: string
+} 
+```
+## Feeders
+### Collections
+* [Pets](#pets)
+### Document Fields
+```ts
+{
+  id: string,
+  modelNumber: string,
+  name: string
+}
+```
+## Notifications
+### Document Fields
+```ts
+{
+  id: string,
+  message: string,
+  time: timestamp
+}
+```
+## Pets
+### Collections
+* [Schedule](#schedule)
+### Document Fields
+```ts
+{
+  RFID: string,
+  age: number,
```

Comment:
Should we replace this with a date of birth so that we can calculate the age and not have to update the database?

Reply:
I think this is a very good call. I will make the adjustments.